SPEC.md

```diff
@@ -190,8 +190,8 @@ In this case, the executor will bind mount the host's `/opt/tenant1/work` direct
 
 #### Network Setup
 
-A Pod must have a [layer 3](http://en.wikipedia.org/wiki/Network_layer) (commonly called the IP layer) network interface, which can be instantiated in any number of ways (e.g. veth, macvlan, ipvlan, device pass-through).
-The network interface MUST be configured with an IPv4/IPv6 address that is reachable from other pods.
+A Pod must have a loopback network interface and one or more [layer 3](http://en.wikipedia.org/wiki/Network_layer) (commonly called the IP layer) network interfaces, which can be instantiated in any number of ways (e.g. veth, macvlan, ipvlan, device pass-through).
+Each network interface MUST be configured with one or more IPv4 and/or IPv6 addresses.  Non-loopback interfaces must have addresses that are reachable from other pods.
```

Comment:
The requirement to have non-loopback interfaces have IP's that are reachable from other pods is both too vague and too strong. It's vague since it does not specify which other pods. appc spec does not talk about clusters so it's unclear about the locus of pods here. It's too strong since we may want to have a pod with two non-loopback interfaces. One of them to talk to other pods and another to talk to the Internet. The second interface would violate the proposed language.